.github/workflows/wheels.yml

```diff
@@ -196,7 +192,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ 'ubuntu-latest', 'windows-latest', 'macos-latest' ]
-        python-version: ['3.7', '3.8', '3.9','3.10', '3.11-dev']
+        python-version: ['3.8', '3.9','3.10', '3.11-dev']
```

Comment:
Any idea on when we could use plain '3.11' instead of '3.11-dev' ?

Reply:
not really - the way i understood this, the "final" version of python needs to be in https://github.com/actions/python-versions/blob/main/versions-manifest.json.
considering 3.11 stable has only been released yesterday, i assume it'll still take a few days - but then changing to "3.11" is absolutely the right approach!